clouddriver-web/src/test/groovy/com/netflix/spinnaker/clouddriver/controllers/OperationsControllerSpec.groovy

```diff
@@ -69,11 +68,17 @@ class OperationsControllerSpec extends Specification {
     }
   }
 
+  static class MockGoogleDeployDescription implements DeployDescription {
+  }
+
+  static class MockAmazonDeployDescription implements DeployDescription {
+  }
+
   @Shared
   def googlePreProcessor = new AtomicOperationDescriptionPreProcessor() {
```

Comment:
Nothing about this is related to google, right? If not, I'd replace it with `testPreProcessor` or something similar. Same with any other mention of a specific cloud provider.

Reply:
Renamed types and variables to remove any appearance of a reference to Amazon or Google implementations.